test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -96,6 +96,16 @@ describe('Client Side Encryption (Legacy)', function () {
       return !isSkippedTest;
     }
 
+    // TODO(NODE-5686): add CSOT support to FLE
+    if (
+      [
+        'timeoutMS applied to listCollections to get collection schema',
+        'remaining timeoutMS applied to find to get keyvault data'
+      ].includes(description)
```

Comment:
https://spruce.mongodb.com/task/mongo_node_driver_next_rhel8_test_serverless_test_serverless_patch_465ffd97974b40e2bf2608907455c00c58c59cff_66705e50f4a35b0007bdeb13_24_06_17_16_03_31?execution=0&sortBy=STATUS&sortDir=ASC

Looks like we're not skipping the test correctly.